### ISSUE TITLE
[iOS] Animations

### ISSUE THREAD

**Issue by wokalski** [react-native-community/react-native-blur#95]
Follow up to the discussion in #62. 

> Instead of using imperative API we can expose and with animation parameters required. What do you think?

@Kureev  I'm not sure what you mean here. Would you like more options in the imperative API or pass somehow the transition to props? (not sure how)

**Comment by wokalski** [comment]
@Kureev ping

**Comment by Kureev** [comment]
You can use LayoutAnimation and/or Animated to achieve this effect. 

P.S. Sorry for the late response.

**Comment by wokalski** [comment]
Not really. Native blur animation kind of blurs in/out. It's not a fade transition.

**Comment by Kureev** [comment]
Does it look different? I've made my conclusion based on the feedback in this issue: https://github.com/react-native-community/react-native-blur/issues/52

**Comment by Kureev** [comment]
maybe @shahen94 or @aforty can give more insights / provide an example

**Comment by wokalski** [comment]
Yes, it is different think of it as going from blur with radius 0 to x by increasing blur radius vs fading in the blur view. I can provide you with an example if you want.

**Comment by Kureev** [comment]
Yeah, I got what you mean. It sounds like a nice improvement, but I'm not sure if I have enough time to pick it up. Although, I'll try to do my best reviewing a PR implementing this feature.

**Comment by fukemy** [comment]
any sample code?
